tests/test_fernet.py

```diff
@@ -122,6 +122,44 @@ def test_bad_key(self, backend):
         with pytest.raises(ValueError):
             Fernet(base64.urlsafe_b64encode(b"abc"), backend=backend)
 
+    def test_timestamp_ttl_overdue(self, monkeypatch, backend):
```

Comment:
This test doesn't appear to be related to this feature. We should already have this test elsewhere in `test_fernet`.

Reply:
removed duplicate test